### PR TITLE
Add cids and update ipfs project 

### PR DESCRIPTION
**Pull request by 2color**
- fix: update ipfs website
- feat: add cids to list of projects
- remove inactive NamChain project

### CODE REVIEW

**Review by vmx — APPROVED** [review]
Though I'd like to see an approval from @rvagg before merging.

File: content/index.md
Comment (bumblefudge) [inline content/index.md:69]:
```suggestion
		<a href="https://github.com/multiformats/cid">CIDs</a>
```

File: content/index.md
Comment (bumblefudge) [inline content/index.md:70]:
```suggestion
		- a self-describing content-addressed identifier scheme for distributed systems, used to address files in IPFS and systems interoperable with it, as well as in other systems like IPLD, where it is the primary way of expressing links between documents.
```

Comment (bumblefudge) [reply content/index.md:70]: (probably more verbose than it makes sense to be here, but just proposing the maximally-detailed version of the summary of what a CID is :D )

**Review by bumblefudge — APPROVED** [review]
LGTM

**Comment by bumblefudge** [comment]
@ramagururadhakrishnan the project you PR'd in seems inactive so we are removing it for the time being, but get in touch with me or @damedoteth if you would like it to be listed here or elsewhere in the community's various downstream trackers!

**Comment by rvagg** [comment]
^ nice, I agree with @bumblefudge's suggestions

File: content/index.md
Comment (2color) [reply content/index.md:70]: Great idea!

**Comment by 2color** [comment]
Whoever has write access, please merge :)

**Pull request merged**
